src/App.tsx

```diff
@@ -5,21 +5,22 @@ import { Routes, Route, Link } from 'react-router-dom'
 import { CollectionProvider } from 'hooks'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { ReactQueryDevtools } from 'react-query/devtools'
+import { StyledNav, StyledNavLink } from './styles'
 const queryClient = new QueryClient()
 
 const App = () => {
   return (
     <div className="App">
-      <nav className="bg-gradient-to-r from-blue-300 to-purple-300 py-16">
-        <ul className="m-0 list-none">
-          <Link className="nav-button" to="/">
-            My Collection
+      <StyledNav>
```

Comment:
Never named anything Styled-

SInce styled-components scopes things to 1 file, I would name it whatever makes sense semantically in the component. It's fine to re-use names too (for example, I use the name "Container" a lot for top level divs)